events/bookmark_events.txt

```diff
@@ -0,0 +1,600 @@
+			}
+			trigger_event = {
+				id = bookmark.0004
+				days = 1
+			}
+			#this triggers the Uthred event
+			trigger_event = {
+				id = uhtred_events.0001
+				days = 1
+			}
+		}
+	}
+}
+
+bookmark.0004 = { #by Mathilda Bjarnehed
+	type = character_event
+	title = bookmark.0004.t
+	desc = {
+		first_valid = {
+			triggered_desc = {
+			    trigger = { #You did not know they had killed your family-member
+					scope:prisoner = {
+						any_secret = {
+	    					secret_type = secret_murder
+	    					secret_target = scope:dead
+	    					NOT = { any_secret_knower = { this = root } }
+	    				}
+					}
+			    }
+			    desc = bookmark.0004.desc_unknown
+			}
+			desc = bookmark.0004.desc
+		}
+	}
+	theme = dungeon
+	left_portrait = {
+		character = scope:prisoner
+		animation = fear
+	}
+	lower_right_portrait = scope:dead
+
+	trigger = {
+		NOT = {
+			any_in_list = {
+				list = captured_rivals
+				this = scope:prisoner
+			}
+		}
+	}
+	
+	immediate = {
+		play_music_cue = "mx_cue_murder"
+		if = {
+			limit = {
+				scope:prisoner = {
+					any_secret = {
+    					secret_type = secret_murder
+    					secret_target = scope:dead
+    					NOT = { any_secret_knower = { this = root } }
+    				}
+				}
+			}
+			scope:prisoner = {
+				random_secret = {
+					limit = {
+						secret_type = secret_murder
+						secret_target = scope:dead
+						NOT = { any_secret_knower = { this = root } }
+					}
+					reveal_to = root
+				}
+			}
+		}
+	}
+
+	#Blood eagle
+	option = {
+		name = bookmark.0004.a
+		
+		add_dread = medium_dread_gain
+		add_prestige = medium_prestige_gain
+		scope:prisoner = {
+			death = {
+				death_reason = death_execution_blood_eagle
+				killer = root
+			}
+		}
+		scope:dead = {
+			every_close_family_member = {
+				custom = bookmark.0004.a.custom
+				limit = { NOT = { this = root } }
+				add_opinion = {
+					modifier = pleased_opinion
+					opinion = 30
+					target = root
+				}
+			}
+		}
+		execute_opinion_effect = { VICTIM = scope:prisoner EXECUTIONER = root }
+
+		ai_chance = {
+			base = 50
+			modifier = {
+				scope:dead = character:163109 #Lodbrok
+				add = 1000
+			}
+		}
+	}
+
+	#Sacrifice to Odin
+	option = {
+		name = bookmark.0004.b
+		trigger = { faith = { has_doctrine_parameter = human_sacrifice_active } }
+
+		add_piety = major_piety_gain
+		scope:prisoner = {
+			death = {
+				death_reason = death_execution_blood_eagle
+				killer = root
+			}
+		}
+		scope:dead = {
+			every_close_family_member = {
+				custom = bookmark.0004.a.custom
+				limit = { NOT = { this = root } }
+				add_opinion = {
+					modifier = pleased_opinion
+					opinion = 30
+					target = root
+				}
+			}
+		}
+		execute_opinion_effect = { VICTIM = scope:prisoner EXECUTIONER = root }
+
+		ai_chance = {
+			base = 50
+		}
+	}
+	
+	#I have other plans...
+	option = {
+		name = bookmark.0004.c
+		
+		ai_chance = {
+			base = 0
+			modifier = {
+				ai_vengefulness <= medium_negative_ai_value
+				add = 100
+			}
+		}
+	}
+}
+
```

Comment:
Will cut code at this point, just noticed the rest of it is not related to Northmen events.